### PR TITLE
Improve readability

### PR DESCRIPTION
**Pull request by andresionek91**
Improve readability of code and fix bad coding patterns.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,24 +3,37 @@
...
+python3 -m virtualenv env
+source env/bin/activate
 ```
 
+2. Install all dependencies by running:
+```
+pip install -r requirements.txt
+```
+
+3. Use the template_local.env to set the environmental variables to a file called
```
Comment (joaodubas) [inline README.md]:
```suggestion
3. Use the template_local.env to set the environment variables to a file called
```

File: rum/app.py [hunk 2]
```diff
@@ -10,41 +11,40 @@ def __init__(self, username, email, access_level='read_only', send_mail=True):
         self.access_level = access_level
         self.sendmail = send_mail
         self.redshift = Redshift()
+        self.check_read_only_exists = open('sqls/check_read_only_exists.sql').read()
+        self.create_user = open('sqls/create_user.sql').read()
+        self.remove_user = open('sqls/remove_user.sql').read()
+        self.add_user_read_only_group = open('sqls/add_user_read_only_group.sql').read()
```
Comment (joaodubas) [inline rum/app.py]: You could create a helper function to read these files. Also use the context manager form of the `open` function to ensure that opened file descriptors area closed. Suggestion:

```python
def read_sql_file(filename: str) -> str:
    with open(filename) as fd:
        return fd.read()


class RedshiftUserManagement:
    def __init__(self, **kwargs):
        ...
        self.check_read_only_exists = read_sql_file('sqls/check_read_only_exists.sql')
        ...
```

File: rum/app.py [hunk 3]
```diff
@@ -10,41 +11,40 @@ def __init__(self, username, email, access_level='read_only', send_mail=True):
...
             try:
                 response = sendgrid.send_usercreated_mail(self.username, password)
-            except Exception as error:
+            except UnauthorizedError as error:
                 raise Exception('If you do not wish to send email to users, '
-                                'then set RedshiftUserManagement(send_mail=False).'
+                                'then set RedshiftUserManagement(send_mail=False). '
                                 'Otherwise, set the correct environmental variables.', error)
             if response >= 300:
                 print('Failed to send email')
```
Comment (joaodubas) [inline rum/app.py]: You could create a method to send an email and invoke it here.

Comment (andresionek91) [reply rum/app.py]: added this exception and if clause to send_mail.py

File: rum/app.py (hunk 2 above)
Comment (andresionek91) [reply rum/app.py]: Great! Thanks for noticing that. Created the function under utils folder.

**Review by joaodubas — APPROVED** [review]

**Pull request merged**